packages/react-components/react-theme/src/alias/darkColor.ts

```diff
@@ -37,11 +37,12 @@ export const generateColorTokens = (brand: BrandVariants): ColorTokens => ({
   colorBrandForeground1: brand[100], // #2899f5 Global.Color.Brand.100
   colorBrandForeground2: brand[110], // #3aa0f3 Global.Color.Brand.110
   colorNeutralForeground1Static: grey[14], // #242424 Global.Color.Grey.14
-  colorNeutralForegroundInverted: white, // #ffffff Global.Color.White
-  colorNeutralForegroundInvertedHover: white, // #ffffff Global.Color.White
-  colorNeutralForegroundInvertedPressed: white, // #ffffff Global.Color.White
-  colorNeutralForegroundInvertedSelected: white, // #ffffff Global.Color.White
+  colorNeutralForegroundInverted: grey[14], // #242424 Global.Color.Grey.14
+  colorNeutralForegroundInvertedHover: grey[14], // #242424 Global.Color.Grey.14
+  colorNeutralForegroundInvertedPressed: grey[14], // #242424 Global.Color.Grey.14
```

Comment:
According to design spec, these 4 aliases should change in teamsDark as well.

Reply:
<img width="1506" alt="image" src="https://user-images.githubusercontent.com/9615899/183058374-f3916c19-a928-43f8-a41f-ec11bf92e34b.png">